A72-open-telemetry-tracing.md

```diff
@@ -36,98 +38,125 @@ and OpenCensus's binary format for (de)serialization. The header name is unique
 from other census library propagators to differentiate with the application’s 
 tracing instrumentation.
 
-The following tracing information during an RPC lifecycle should be captured:
+### Related Proposals and Documents:
+* [gRFC L29: C++ API Changes for OpenCensus Integration][L29]
+* [gRFC A45: Exposing OpenCensus Metrics and Tracing for gRPC retry][A45]
+* [Microservices observability overview][grpc-observability-public-doc]
+* [gRFC A66: OpenTelemetry Metrics][A66]
+
+## Proposal
+### gRPC OpenTelemetry Tracing API
+We will add tracing functions in grpc-open-telemetry plugin, along with OpenTelemetry
+metrics [gRFC A66][A66]. Languages will keep using gRPC infrastructures, e.g. interceptor and
+stream tracer to implement the feature, the same as Census.
+
+The OpenTelemetry API will coexist with the OpenCensus API. Only one
+`grpc-trace-bin` header will be sent for a single RPC as long as only one of
+OpenTelemetry or OpenCensus is enabled for the channel. The APIs to enable and
+configure OpenTelemetry tracing are different among languages due to different 
+underlying infrastructures.
+
+#### Java
+In Java, it will be part of global interceptors, so that the interceptors are
+managed in a more sustainable way and user-friendly. As a prerequisite, the stream
+tracer factory API will be stabilized. OpenTelemetryModule will be created with
+an OpenTelemetry API instance passing in for necessary configurations.
+Users can also rely on SDK autoconfig extension that configures the sdk object
+through environmental variables or Java system properties, then pass the 
+obtained sdk object to gRPC.
+
+```Java
```

Comment:
This java section shows an example of how to use the API rather than defining the API itself, whereas the C++ section below does the opposite.  Let's pick one of the two approaches and be consistent.  @yashykt

Reply:
Good call, fixed. I added the API itself.